pkg/subctl/operator/submarinerop/deployment/ensure.go

```diff
@@ -68,6 +68,10 @@ func Ensure(kubeClient kubernetes.Interface, namespace, image string, debug bool
 							Image:           image,
 							Command:         command,
 							ImagePullPolicy: imagePullPolicy,
+							SecurityContext: &v1.SecurityContext{
+								RunAsNonRoot:             boolPointer(true),
+								AllowPrivilegeEscalation: boolPointer(false),
```

Comment:
It turns out there’s a K8s library we can use, `k8s.io/utils/pointer`; see https://github.com/submariner-io/submariner-operator/pull/1859